pkg/sfu/videolayerselector/dependencydescriptor.go

```diff
@@ -2,39 +2,37 @@ package videolayerselector
 
```

Comment:
Biggest changes in this file. Sorry, the diff is hard to read. To summarise
- Check if a frame is decodable
- Check if decode chain is integral
- Based on those, switch layers or drop frames